packages/twenty-front/src/modules/ui/input/components/internal/date/components/InternalDatePicker.tsx

```diff
@@ -321,6 +304,9 @@ export const InternalDatePicker = ({
   isDateTimeInput,
   keyboardEventsDisabled,
   onClear,
+  isRelative,
+  onRelativeDateChange,
+  relativeDateFilterValue,
 }: InternalDatePickerProps) => {
   const internalDate = date ?? new Date();
 
```

Comment:
DatePicker seems entirely broken for me (for example I can't edit the value of a date field)

<img width="595" alt="Screenshot 2024-09-23 at 11 24 14" src="https://github.com/user-attachments/assets/8ce35f36-1f6a-4ce9-835b-775f389f0f97">